hw/xilinx/Makefile

```diff
@@ -16,31 +16,51 @@ XILINX_PROJECT_REPORTS_DIR ?= ${XILINX_PROJECT_BUILD_DIR}/reports
 
 # List of the Xilinx IPs to build and import in the design
 # Parsing from directory ips/
+XILINX_COMMON_IP_LIST 	= $(shell basename --multiple ${XILINX_IPS_ROOT}/common/xlnx_*)
+XILINX_HPC_IP_LIST 		= $(shell basename --multiple ${XILINX_IPS_ROOT}/hpc/xlnx_*) 
+XILINX_EMBEDDED_IP_LIST = $(shell basename --multiple ${XILINX_IPS_ROOT}/embedded/xlnx_*) 
 
-XILINX_COMMON_IP_LIST = $(shell basename --multiple ${XILINX_IPS_ROOT}/common/xlnx_*)
-XILINX_HPC_IP_LIST = $(shell basename --multiple ${XILINX_IPS_ROOT}/hpc/xlnx_*)
-XILINX_EMBEDDED_IP_LIST = $(shell basename --multiple ${XILINX_IPS_ROOT}/embedded/xlnx_*)
+# List of the Custom IPs to build and import in the design
+# Parsing from directory ips/
+CIP_COMMON_IP_LIST 		= $(shell if ls ${XILINX_IPS_ROOT}/common/cip_* 1>/dev/null 2>&1; then basename --multiple ${XILINX_IPS_ROOT}/common/cip_*; else echo ""; fi)
```

Comment:
Same as above, I suggest COMMON_IP_LIST etc

Reply:
CUSTOM_HPC_IP_LIST 
CUSTOM_EMBEDDED_IP_LIST 
etc..

Reply:
Try to fix this with `basename` flags, not with if/then and redirect to /dev/null etc